aim/web/ui/src/pages/Home/components/HomeLeft/QuickLinks/QuickLinks.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Tooltip } from '@material-ui/core';
+
+import { Icon, Text } from 'components/kit';
+import ListItem from 'components/kit/ListItem/ListItem';
+
+import { encode } from 'utils/encoder/encoder';
+
+import './QuickLinks.scss';
+
+const linkItems: string[] = ['active', 'archived'];
+
+function QuickLinks(): React.FunctionComponentElement<React.ReactNode> {
+  const history = useHistory();
+
+  const onClick: (
+    e: React.MouseEvent<HTMLElement>,
+    value: string,
+    newTab?: boolean,
+  ) => void = React.useCallback(
+    (e: React.MouseEvent<HTMLElement>, value: string, newTab = false) => {
+      e.stopPropagation();
+      e.preventDefault();
```

Comment:
Why we are using preventDefault here? Are the Icon and Text components have a default behavior that we want to prevent?